mplc/contributivity.py

```diff
@@ -1084,16 +1086,21 @@ def federated_SBS_constant(self, the_scenario):
         end = timer()
         self.computation_time_sec = end - start
 
-    def compute_relative_perf_matrix(self, the_scenario):
+    def compute_relative_perf_matrix(self):
 
         # Define proportion of initial and final computation rounds to skip (default 10% each)
         init_comp_rounds_skipped = 0.1
         final_comp_rounds_skipped = 0.1
 
         # Fetch score matrices from computation and scenario characteristics
-        multi_partner_learning = the_scenario.mpl
-        score_matrix_collective_models = multi_partner_learning.score_matrix_collective_models[:, 1:]
-        score_matrix_per_partner = multi_partner_learning.score_matrix_per_partner
+        multi_partner_learning = self.scenario.mpl
+        score_matrix_collective_models = multi_partner_learning.history.history['model']['val_accuracy'][:, 1:]
```

Comment:
line 1096 triggers an error in line 1109. With minibatch 2 and 10 epoch `score_matrix_collective_models` should be of size 20 but was of size 10.